### PR TITLE
Add basic tests

### PR DESCRIPTION
**Pull request by jules2689**
@kevintpeng

### CODE REVIEW

File: js/comms.js [hunk 1]
```diff
@@ -42,18 +41,11 @@ class Comms {
...
 
-  sendDataToPeer(peer, type, data) {
-    if (this.peers.indexOf(peer) == -1 || peer == this.id) {
-      return
-    }
-
-    data['type'] = type;
-    var stringData = JSON.stringify(data);
-
+  sendDataToPeer(id, type, data) {
```
Comment (jules2689) [inline js/comms.js:44]: mostly ignore this cause we're replacing peer.js

**Comment by jules2689** [comment]
We can setup circle ci to run this all for free btw. https://circleci.com/gh/kevintpeng/paxos-video/edit

File: test/paxos_test.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+      var log = paxos.comms.sendDataToPeers['id'];
+      assert.equal('RESPONSE', log['type']);
+      expect(log['data']['accept']).to.be.true;
+      expect(log['data']['state']).to.be.null;
+      assert.equal(2, log['data']['seq']);
+      assert.equal(2, paxos.accepted_seq);
+    });
+
+    it('it should reject if less than than current sequence', function() {
```
Comment (kevintpeng) [inline test/paxos_test.js]: remove double `it` on these? Or is that the idomatic js way to do things?

Comment (jules2689) [inline test/paxos_test.js]: nah, I'm just dumb. Removed them

**Comment by kevintpeng** [comment]
I think one of my comments got lost, but yeah this looks good, we can definitely add more tests for paxos.js but it's good that this is working :)

**Pull request merged**